rust/protocol/src/sealed_sender.rs

```diff
@@ -853,6 +990,21 @@ mod sealed_sender_v2 {
         Ok(result)
     }
 
+    /// Compute an [authentication tag] for the encrypted result `encrypted_message_key` produced by
+    /// [apply_agreement_xor] using a secret derived from the sender's private identity key
+    /// `our_keys` and the recipient's public identity key `their_key`.
+    ///
+    /// As with [apply_agreement_xor], the result of this method called with [Direction::Sending] is
+    /// then encoded into the message produced by
+    /// [super::sealed_sender_multi_recipient_encrypt]. The message recipient can securely derive
+    /// the same output bytes by calling this method using [Direction::Receiving], their private key
+    /// `our_keys`, the sender's public key `their_key`, and the ephemeral public key
+    /// `ephemeral_pub_key` which was produced by the sender from [DerivedKeys::e].
+    ///
+    /// *See more information on [authenticated encryption].*
+    ///
+    /// [authentication tag]: https://crypto.stackexchange.com/a/67108
+    /// [authenticated encryption]: https://en.wikipedia.org/wiki/Authenticated_encryption
```

Comment:
I think you could just link to the Wikipedia article to explain "authentication tag" instead of picking a specific StackExchange post. Feels weird to see that in a codebase to explain a generic crypto term.

Reply:
Yes, this makes sense. I think it's because wikipedia doesn't immediately redirect "authentication tag" to "message authentication code" and I didn't do enough looking around. I agree it feels safer to link to wikipedia in general.